### PR TITLE
Blurred video background

### PR DESCRIPTION
**Pull request by jondujaka**
- Generate bigger blurred background image at build time
- Generate the dominant color of this background
- Check the contrast between the dominant color and white text
- Add class to invert colors to controls if the contrast is not readable

Steps to test:

1. Open video focus page /video/[id]
2. You should see a blurred thumbnail filling the page
3. If the image is very light, the text will be black
4. Otherwise, it will be white

### CODE REVIEW

File: netlify/functions/getVideoThumbnail.ts [hunk 1]
```diff
@@ -10,14 +10,34 @@ const getImage = async url => {
...
     .resize(12, 12, { fit: 'inside' })
-    .toFormat(sharp.format.png)
-    .toBuffer();
+    .toFormat(sharp.format.png);
 
-  return buffer.toString('base64');
+  const backgroundSharpObject = await sharp(image)
+    .raw()
+    .ensureAlpha()
+    .resize(1200, 1200, { fit: 'inside' })
```
Comment (Runeii) [inline netlify/functions/getVideoThumbnail.ts:22]: Does it need to be this big, given we're blurring it? Just wondering about how much this adds to page size on projects with many videos.

File: src/components/Video/Video.tsx [hunk 2]
```diff
@@ -1,60 +1,115 @@
-import useSwr from 'swr';
+import React, {
```
Comment (Runeii) [inline src/components/Video/Video.tsx:1]: As we need the video data on the focused video player view, I was able to move VideoContent back to being known as Video.

File: src/components/Video/Video.module.scss [hunk 3]
```diff
@@ -1,6 +1,25 @@
 .frame {
   position: relative;
   overflow: hidden;
+  padding: 0;
```
Comment (Andrew387) [inline src/components/Video/Video.module.scss:4]: Can be removed as 0 is the default and the value doesn't change

Comment (Runeii) [reply src/components/Video/Video.module.scss:4]: Fixed!

**Pull request merged**